pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/institution/DDPInstitutionDao.java

```diff
@@ -61,43 +66,35 @@ public int create(DDPInstitutionDto ddpInstitutionDto) {
 
     @Override
     public int delete(int id) {
-        return 0;
+        SimpleResult simpleResult = DaoUtil.deleteById(id, SQL_DELETE_BY_ID);
```

Comment:
I see ddp_institution used in many join statements outside of this code.  Any impact to those if an institution is deleted after it's been used by one of those "joining" queries?

Reply:
I believe those are all protected by key constraints, if I understand your question correctly.
I did not add testing for SQL_DELETE_BY_ID since I am only using it *in* at test now.